site/app/libraries/Utils.php

```diff
@@ -286,4 +286,33 @@ public static function removeStudentWithId($students, $key, $id) {
         }
         return $students;
    }
+
+   /**
+    * Convert the shorthand byte notation in php.ini to bytes.
+    * E.g : php return_bytes(ini_get('post_max_size'))
+    * Src : https://www.php.net/manual/en/function.ini-get.php
+    * @param string $val
+    * @return int 
+    */
+    public static function return_bytes ($size_str){
+        switch (substr ($size_str, -1)){
+            case 'M': case 'm': return (int)$size_str * 1048576;
+            case 'K': case 'k': return (int)$size_str * 1024;
+            case 'G': case 'g': return (int)$size_str * 1073741824;
+            default: return $size_str;
+        }
+    }
+
+    /**
+    * Convert bytes to a specified format thats human readable 
+    * E.g : MB, 10485760 => 10Mb
```

Comment:
I am not very sure about that but I vaguely remember that `b` denotes `bit` and `B` denotes `byte`. 10485760 bytes is supposed to be 10 megabytes. 

ref: https://www.quora.com/What-is-the-difference-between-Kb-and-KB

Reply:
So maybe not use `ucfirst` below.